src/main/java/com/github/vkorobkov/jfixtures/loader/FixturesLoader.java

```diff
@@ -49,6 +51,7 @@ private boolean isNotConfig(Path file) {
     private Fixture loadFixture(Path file) {
         String name =  getFixtureName(file);
         val baseColumns = config.columns().forTable(name);
+        log.info("Processing table '" + name + "'");
```

Comment:
We probably should move this logging into some place at the `Processor`. `FixturesLoader` does not even loading a fixture - this operation is lazy